docs/io/fits/usage/cloud.rst

```diff
@@ -46,12 +46,12 @@ You can improve the performance for large files by passing the parameters
 Astropy use ``fsspec`` to download only the necessary parts of the FITS file.
 For example:
 
-.. doctest-remote-data::
+.. doctest-requires:: fsspec
 
     >>> from astropy.io import fits
     ...
     >>> # `fits.open` will download the primary header
-    >>> with fits.open(url, use_fsspec=True, lazy_load_hdus=True) as hdul:
+    >>> with fits.open(url, use_fsspec=True, lazy_load_hdus=True) as hdul:  # doctest: +REMOTE_DATA
```

Comment:
if you do it this way, `+REMOTE_DATA` should be repeated on each and every line below. Use the directive instead. If there are any issues with using multiple together, report it upstream to `pytest-doctestplus`

Reply:
Is it possible to use two directives together?  i.e. can I combine these two:
```
.. doctest-requires:: fsspec
.. doctest-remote-data::
```

Reply:
I see that we have a test in doctestplus for the other case, so it is supposed to work. But I can't recall any use case where we actually used multiple directives together.

```
.. doctest-remote-data::
.. doctest-requires:: fsspec
```